gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
Was this part of rerunning the wrapper, or did you have a reason to customize the heap size?

Reply:
It was part of rerunning the wrapper, I didn't mix in any custom changes.

Reply:
Cool, that's what I figured, but used to seem ng more changes in a wrapper script update, so it seemed odd.